vectorstores/pgvector/pgvector.go

```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/tmc/langchaingo/vectorstores"
 )
 
+const (
```

Comment:
Add a comment saying these numbers are arbitrary and should just be different from each other